### PR TITLE
Connection Disconnection 

### LINKED ISSUE #32: Probe: Connection and disconnection
- Probe send a connection request for connecting to C&C Server and send disconnection request for disconnection. 
- Implement through grpc calls

### PR DESCRIPTION
**Pull request by dhruvi29**
# Description
- This PR writes 2 major files, connection_request.py and disconnection_request.py
- The uid received upon connection is stored in `uid.id`. This uid is used while sending disconnection request. 
- The grpc server ip should be editable and hence made a part of the environment variables with this PR 

fixes #32  

## Type of change
- [x] New feature (non-breaking change which adds functionality)
- [x] minor bug fix

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: C&C Server/.env
Comment (ThatUdeshUK) [inline C&C Server/.env:1]: Let's update this to align with out last PR #31

File: Probe/.env
Comment (ThatUdeshUK) [inline Probe/.env:1]: Change the `.env` to match with PR #31

**Review by ThatUdeshUK — CHANGES_REQUESTED** [review]
Nice job. Let's fix the conflicts before we merge this.

File: Probe/.env
Comment (dhruvi29) [reply Probe/.env:1]: resolved :+1:

**Review by ThatUdeshUK — APPROVED** [review]

**Pull request merged**